path.go

```diff
@@ -151,11 +151,11 @@ func readFile(p string) ([]byte, error) {
 	}
 
 	// Re-fetch remote file and create cache
-	pathstr, err := filepath.Rel(globalCacheDir, p)
+	path, err := filepath.Rel(globalCacheDir, p)
```

Comment:
You were able to understand the reason, but you feel that ignoring gostyle to avoid namespace issues that are too general is not a good idea.

I thought that variable names that match the context should be used instead. I reviewed the variable names. What do you think?

Reply:
Nice!! It seems that the idea of "making variable names good and understandable" had disappeared from my mind.